lib/config/validation.ts

```diff
@@ -16,6 +17,15 @@ export interface ValidationResult {
   warnings: ValidationMessage[];
 }
 
+const managerList = getManagerList();
+
+function isManagerPath(parentPath: string): boolean {
+  return (
+    /^regexManagers\[[0-9]+]$/.test(parentPath) ||
+    managerList.includes(parentPath)
```

Comment:
Did I miss something?

Reply:
Now I'm worried that I forgot something else. Mostly I just don't want it configured at top level

Reply:
Then maybe error for top-level and warning for others?

Reply:
I think it's ok as-is